README.md

```diff
@@ -2,7 +2,7 @@
 - The Fenders Perth Meetup website.
 
 ## TODO
-- [ ] Setup components
+- [x] Setup components
```

Comment:
Can just remove this if it's done?

Reply:
Unsure. It was setup as a checkbox so I assumed they wanted it ticked. @amykapernick ?